### ISSUE TITLE
Is requiring the use of a trademark non-free?

### ISSUE THREAD

**Issue by davelab6** [collabmark/collabmark.org#1]
This site doesn't mention that requiring the use of trademarks is non-free, but this idea was recently presented by @pchestek at [FOSDEM 2015](https://archive.fosdem.org/2014/schedule/event/trademark_licenses/) and I wonder if you agree, and if so, suggest stating this on the site.

**Comment by slaporte** [comment]
I agree that requiring a trademark is non-free, depending on the exact requirement. [Freedom Defined](http://freedomdefined.org/Permissible_restrictions#Attribution_of_authors_.2F_No_Endorsement) explains that a copyright license is not free if it requires advertisement or attribution that impedes normal use of a work. We discuss the history of the advertisement clause in the pre-1997 BSD license in [our paper](http://papers.ssrn.com/sol3/papers.cfm?abstract_id=2476779), but we don't go into detail on the site. 

I think this is a good topic to include in a FAQ. I'll add it to our TODO list. Thanks!
